charts/rstudio-connect/values.yaml

```
@@ -333,6 +333,12 @@ config:
   Server:
     Address: http://localhost:3939
     DataDir: /var/lib/rstudio-connect
+  Python:
+    Enabled: true
+    Executable: "/opt/python/3.9.17/bin/python3"
```

There are 2 python installations available on the default image, should we include both?

```suggestion
    Executable:
      - /opt/python/3.9.17/bin/python
      - /opt/python/3.8.17/bin/python
```